principles.md

```diff
@@ -1,51 +1,71 @@
 # The 5 principles of Standard Containers
 
 Define a unit of software delivery called a Standard Container.
-The goal of a Standard Container is to encapsulate a software component and all its dependencies in a format that is self-describing and portable, so that any compliant runtime can run it without extra dependencies, regardless of the underlying machine and the contents of the container.
-
-The specification for Standard Containers is straightforward.
-It mostly defines 1) a file format, 2) a set of standard operations, and 3) an execution environment.
-
-A great analogy for this is the shipping container.
-Just like how Standard Containers are a fundamental unit of software delivery, shipping containers are a fundamental unit of physical delivery.
+The goal of a Standard Container is to encapsulate a software component and
+all its dependencies in a format that is self-describing and portable, so that
```

Comment:
Markdown policy is [one sentence per line][1] (which is where the long lines from the original came from ;).

[1]: https://github.com/opencontainers/specs/blob/v0.1.1/README.md#markdown-style